dev_guide/limits.adoc

```diff
@@ -47,25 +105,38 @@ Per container, the following must hold true if specified:
 |`*Min*`
 |Min[resource] <= container.resources.requests[resource] (required) <= container/resources.limits[resource] (optional)
 
+If the configuration defines a `min` CPU, then the request value must be greater
+than the CPU value. A limit value does not need to be specified.
+
 |`*Max*`
 |container.resources.limits[resource] (required) <= Max[resource]
 
+If the configuration defines a `max` CPU, then you do not need to define a
+request value, but a limit value does need to be set that matches the maximum
```

Comment:
matches the maximum is confusing... maybe just say satisfies the maximum CPU constraint?